packages/components/src/tabs/styles.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import { COLORS } from '../utils';
+import { space } from '../ui/utils/space';
```

Comment:
The `space` utils have been relocated after https://github.com/WordPress/gutenberg/pull/54922